### DISCUSSION TITLE
Howto config using non tech language

### DISCUSSION THREAD

**Discussion by sonofcy** [easytarget/esp32-cam-webserver#257] [Q&A]
I want my laptop that is connected to my router to be able to be connected to the webserver camera. In other words my laptop still has full internet access AND access to the camera webserver. 
I do NOT understand the lingo AP, ST, portal etc, I just need to know what variables to modify and examples of contents.
Q why is this not using WiFiManager that works great for me when I added it to the old esp32cam-webserver

**Comment by easytarget** [comment]
Sorry if that gave you a headache; I usually try to avoid acronyms, or at least make sure I clearly define them whn I first use them; obviously failed on this occasion, :frown:

Wifimanager was always a good suggestion; but never implemented.
